packages/docs/docs/tutorials/importing-sample-data.md

```diff
@@ -0,0 +1,61 @@
+
+## Browse and Understand Data
+
+The dataset provided demonstrates how to represent many common data types. Click on the links provided to see the resources.
+
+| Data Type                                                                                                                                                                                                                                                                                                                                                                                   | Browse on Medplum                                                                                                                                                                                                                                                                                                                                      | Related Documentation                                                                                                                                                                        |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Allergies and Intolerances                                                                                                                                                                                                                                                                                                                                                                  | [AllergyIntolerance](https://app.medplum.com/AllergyIntolerance?_count=20&_fields=_lastUpdated,patient,clinicalStatus,verificationStatus,type,category,code,recordedDate&_offset=0&_sort=-_lastUpdated)                                                                                                                                                | [Charting](/docs/charting)                                                                                                                                                                   |
```

Comment:
This is a great table, but it would be more helpful if it was sorted in "commonly-used" order rather than alphabetical order. My proposed order: 

1. Patient
2. Practitioner
3. Problems
4. Vitals
5. Labs
6. Medications
7. Immunizations
8. Allergies
9. Encounter
10. Notes
11. Procedures
12. Smoking status
13. Implantable Device